src/chains/config/fantom.config.ts

```diff
@@ -19,7 +19,7 @@ export class Fantom extends Chain {
       fantomTokensConfig,
       fantomSetts,
       rpc[Network.Fantom],
-      new FantomStrategy(Object.keys(fantomTokensConfig)),
+      new BaseStrategy(Network.Fantom, Object.keys(fantomTokensConfig)),
```

Comment:
we have removed all the specific strategies since all these custom functions just injected chains, we will move toward removing the strategy specific look ups to enable sorting and batching on these price checks